scripts/upgrade.d/upgrade.last.sh

```diff
@@ -1,16 +1,25 @@
-gitlab_version="12.9.2"
+gitlab_version="12.10.0"
 
-gitlab_x86_64_source_sha256="781d21de10c4b88582d25af19cd3d85d9618a995f930d1954ea9dc0fa76d7ea9"
+# sha256sum found here: https://packages.gitlab.com/gitlab
 
-gitlab_arm_source_sha256="09eb415d2e55af64294606e25cfebd508c8cc59218475ed26e99f6baccfe1218"
+if [ "$(lsb_release -sc)" = "buster" ]
+then
+	gitlab_x86_64_source_sha256="01d8d1dd2e4ed9a378317f55ab46174d6bd77e4eb1be32d87fe7911da8f65381"
+else
+	gitlab_x86_64_source_sha256="ed24c9bd072b6f15f7c73367df04acba3e64ee53bb3562f25e768a2dda533ca2"
+fi
+
+gitlab_x86_64_debian_verion="$(lsb_release -sc)"
+
+gitlab_arm_source_sha256="5d4b1d76997de08e9707e5cb17445a38ed832bd9995f31a5af0b87134b7f0834"
 
 gitlab_filename="gitlab-ce-${gitlab_version}.deb"
 
 # Action to do in case of failure of the package_check
 package_check_action() {
 	local sysctl_file="$final_path/embedded/cookbooks/package/resources/gitlab_sysctl.rb"
-	if [ ! -f "$sysctl_file" ]; then
-		sysctl_file="$final_path/embedded/cookbooks/package/resources/sysctl.rb"
-	fi
-	ynh_replace_string --match_string="command \"sysctl -e --system\"" --replace_string="command \"sysctl -e --system || true\"" --target_file=$sysctl_file
+	ynh_replace_string --match_string="command \"sysctl -e \(.*\)\"" --replace_string="command \"sysctl -e \1 || true\"" --target_file=$sysctl_file
+	
+	sysctl_file="/opt/gitlab/embedded/cookbooks/package/recipes/sysctl.rb"
+	ynh_replace_string --match_string="command \"sysctl -e \(.*\)\"" --replace_string="command \"sysctl -e \1 || true\"" --target_file=$sysctl_file
```

Comment:
It's not specific to package_check anymore?

Reply:
No, with that fix you can install it on lxc (scaleway for example).

Here the related issue https://github.com/YunoHost-Apps/gitlab_ynh/issues/66

Reply:
Ok, so finally you're just ignoring the error if it happens with sysctl.
And it does work anyway ?
Ok, that's fine for me.

Reply:
Yes, it's working, but not as fast as possible if I well understand it. It's better not to ignore these errors, but in an environment where the kernel parameters are read-only, I couldn't find any other solution.